scriptworker/task.py

```diff
@@ -229,16 +230,15 @@ def guess_content_type(path):
     }
     args = [get_task_id(context.claim_task), get_run_id(context.claim_task),
             target_path, payload]
+    _compress_if_text_plain(path, payload['contentType'])
```

Comment:
Good news! No need to modify download_cot(). [aiohttp gunzip files for us](http://aiohttp.readthedocs.io/en/stable/client.html#binary-response-content) when downloaded (behavior tested locally).